src/renderer/src/routes/pages/Send/SendRecipient.tsx

```diff
@@ -81,7 +81,8 @@ export const SendRecipient = ({
   }
 
   const handleChangeAmount = (event: ChangeEvent<HTMLInputElement>) => {
-    const value = NumberHelper.formatString(event.target.value, recipient.token?.token.decimals)
+    const value = NumberHelper.formatString(event.target.value, recipient.token?.token?.decimals ?? 18, 24)
```

Comment:
It is not a good approach to pass 18 when decimals are undefined, as the token can only have 8 and this will break the send. I recommend doing a conditional first to check if there are decimals.
```suggestion
    if (recipient.token?.token?.decimals === undefined) return
    const value = NumberHelper.formatString(event.target.value, recipient.token.token.decimals, 24)
```